### PR TITLE
Update SIG ContribEx teams

### PR DESCRIPTION
**Pull request by mrbobbytables**
This is a first stab at cleaning up the various teams, several aren't in use anymore and others haven't been touched in forever :x

/hold for review
/assign @alisondy @cblecker @nikhita

### CODE REVIEW

File: config/kubernetes/sig-contributor-experience/teams.yaml [hunk 1]
```diff
@@ -88,51 +78,8 @@ teams:
     - justaugustus
     - lachie83
     - LappleApple
-    - mattfarina
-    privacy: closed
-  sig-contributor-experience:
```
Comment (cblecker) [inline config/kubernetes/sig-contributor-experience/teams.yaml:93]: Completely removing this?

File: config/kubernetes/sig-contributor-experience/teams.yaml [hunk 2]
```diff
@@ -88,51 +78,8 @@ teams:
...
-    - philips
-    - Phillels
-    - pigmej
-    - pwittrock
-    - thockin
-    - xiang90
     privacy: closed
-    previously:
-      - sig-contributor-experience-misc-use-only-as-a-last-resort
     teams:
```
Comment (cblecker) [inline config/kubernetes/sig-contributor-experience/teams.yaml:82]: This messes up the sub teams, doesn't it?

**Review by cblecker — CHANGES_REQUESTED** [review]

File: config/kubernetes/sig-contributor-experience/teams.yaml (hunk 2 above)
Comment (mrbobbytables) [reply config/kubernetes/sig-contributor-experience/teams.yaml:82]: You'r right it does -_- could move them up a level, on a side note - it was not caught by our tests :x

File: config/kubernetes/sig-contributor-experience/teams.yaml (hunk 1 above)
Comment (mrbobbytables) [reply config/kubernetes/sig-contributor-experience/teams.yaml:93]: I need to fix the formatting, but probably? the group is very stale. The instances that I know it where it was used, it was more or less targeted at the leads.

ref: https://github.com/search?o=desc&q=org%3Akubernetes+team%3Akubernetes%2Fsig-contributor-experience&s=created&type=Issues

**Pull request closed without merging**